library/src/main/java/org/codice/security/sign/SimpleSign.java

```diff
@@ -241,6 +252,76 @@ public boolean validateSignature(
     }
   }
 
+  public void validateSignature(Signature signature) throws SignatureException {
```

Comment:
❗️ Should add a comment about further researching this method. From what I've seen it looks incomplete.

For instance: If no `KeyInfo` is passed in, this method errors out. The spec says that if no `KeyInfo` exists then we should try and decrypt it with their signing key.

Reply:
There is a card to investigate that.